testing/run-scripts/image_make.sh

```diff
@@ -51,7 +55,17 @@ TMP_DIR=`mktemp -d`
 
 cp -R ${BASH_SOURCE%/*}/../integration/test $TMP_DIR/test
 
-cat <<EOF > $TMP_DIR/Dockerfile
+if [ "$DOCKER" = true ]
+then
+  cat <<EOF > $TMP_DIR/Dockerfile
+FROM soycode/uproxy-zork
+
+RUN apt-get -qq update
+RUN apt-get -qq upgrade
```

Comment:
That's a very interesting question. I'm tempted to leave it as-is because we basically don't care about updates besides security, which unattended-upgrades will take care of.